exonum-java-binding/time-service/README.md

```diff
@@ -0,0 +1,52 @@
+# Time oracle
+
+Java Binding Time Oracle provides proxy for [Exonum Time Service][exonum-time].
+This service allows to determine time, import it from the external world to the blockchain and keep
+its current value in the blockchain.
+
+## Usage
+
+Include `exonum-time-oracle` as a dependency in your `pom.xml`:
+
+``` xml
+    <dependency>
+      <groupId>com.exonum.binding</groupId>
+      <artifactId>exonum-java-binding-time-service</artifactId>
+      <version>0.5-SNAPSHOT</version>
+    </dependency>
+```
+
+To use the time oracle, it needs to be enabled. To do that, include its name `time` in
+`ejb_app_services.toml` configuration file in the EJB App's directory with the following content:
+
+```toml
+services = ["time"]
+```
+
+See more information on built-in services [here][built-in-services].
+
+Below is an example of a service method that uses time oracle to return the consolidated time. Note
+that at the time when a new blockchain is launched, the consolidated time is unknown until the
+transactions from at least two thirds of validator nodes are processed. In that case the result
+will not contain a value. Validator nodes send the transactions after the commit of each block.
```

Comment:
I would adjust the paragraphs as follows:

Below is an example of a service method that uses time oracle to return the consolidated time. Note
that at the time when a new blockchain is launched, the consolidated time is unknown until the
transactions containing time values from at least two thirds of validator nodes are processed. Before that, the method will return a result with zero value. Validator nodes send the transactions after the commit of each block.

Reply:
I don't think that `zero value` is correct here 🤔 It's not the same as the absence of value

Reply:
empty value will also work for an optional.

I'd also add, as suggested, "… containing time values" so that it is clear which transactions.

Reply:
use "empty value" as Dmitry says